app/models/manageiq/providers/vmware/manager_mixin_auth.rb

```diff
@@ -0,0 +1,50 @@
+module ManageIQ::Providers::Vmware::ManagerMixinAuth
```

Comment:
Minor, but the pattern is to put Mixin at the end of the name...that is, this should probably be `ManagerAuthMixin`